src/review.rs

```diff
@@ -24,14 +24,19 @@ pub struct Review {
 
 /// Metadata for a single review. Stored as dotfile next to user-facing review file
 #[derive(Serialize, Deserialize, Debug)]
-struct ReviewMetadata {
+pub(crate) struct ReviewMetadata {
```

Comment:
All this code is in the same crate and we're not exporting a library so I think it should be fine to leave as `pub` rather than `pub(crate)`